_posts/2015-01-03-Update-December-2014.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: default
+title: Update December 2014
+---
+
+Welcome to the 6th monthly PHPCR news update and the last of 2014.
+
+### CMF Meetup
+
+On the 12th of Decembed the CMF team metup in Würzburg, Germany and worked on
+Jackalope in amongst other things. We also discussed the next iteration of
+Jackalope, "Jackalope 2" - which would aim to be more modular and more
+performant.
+
+We hope to hold another meetup early this year.
```

Comment:
you could link the cmf mailinglist and tell people to sign up there if interested.